modules/apps/object/object-service/src/main/java/com/liferay/object/internal/search/spi/model/index/contributor/ObjectDefinitionModelDocumentContributor.java

```diff
@@ -35,10 +41,34 @@ public class ObjectDefinitionModelDocumentContributor
 	public void contribute(
 		Document document, ObjectDefinition objectDefinition) {
 
-		document.addLocalizedKeyword(
-			"localized_label", objectDefinition.getLabelMap(), true, true);
-		document.addText(Field.NAME, objectDefinition.getShortName());
-		document.remove(Field.USER_NAME);
+		try {
+			ObjectFolder objectFolder =
+				_objectFolderLocalService.getObjectFolder(
+					objectDefinition.getObjectFolderId());
+
+			document.addKeyword(
+				"objectFolderExternalReferenceCode",
+				objectFolder.getExternalReferenceCode());
```

Comment:
@murilostodolni since we're not going to follow this approach of filtering by ERC, we can get rid of this peace of code for now.

Reply:
yes, I dropped commit.